schema/whats-new-panel.schema.json

```diff
@@ -35,41 +41,26 @@
   },
   "properties": {
     "id": {
-      "type": "string",
+      "$ref": "#/definitions/validStringOrID",
       "description": "Message identifier"
     },
     "content": {
       "type": "object",
       "properties": {
         "layout": {
-          "type": "string",
+          "$ref": "#/definitions/validStringOrID",
           "description": "Attribute used for different groups of messages from the same provider"
         },
         "body": {
           "description": "This can be a reference to a localized string in Firefox or just a plain string.",
-          "oneOf": [
-            {
-              "type": "string",
-              "description": "Description message of the addon."
-            },
-            {
-              "type": "object",
-              "properties": {
-                "string_id": {
-                  "type": "string",
-                  "description": "Id of string to localized addon description"
-                }
-              },
-              "required": ["string_id"]
-            }
-          ]
+          "$ref": "#/definitions/stringOrStringID"
```

Comment:
Hmm, what if `body` is an object with a `string_id`? Perhaps you wanted to just replace the arm for the `string` type?